app/src/renderer/vuex/modules/ledger.js

```diff
@@ -98,16 +98,13 @@ export default () => {
       }
     },
     async getLedgerCosmosVersion({ commit, state }) {
-      let response
-      try {
-        response = await state.cosmosApp.get_version()
-        actions.checkLedgerErrors(response)
-        const { major, minor, patch, test_mode } = response
-        const version = { major, minor, patch, test_mode }
-        commit(`setCosmosAppVersion`, version)
-      } catch (error) {
-        Sentry.captureException(error)
-        commit(`setLedgerError`, error)
+      const response = await state.cosmosApp.get_version()
+      actions.checkLedgerErrors(response)
+      const { major, minor, patch, test_mode } = response
+      const version = { major, minor, patch, test_mode }
+      commit(`setCosmosAppVersion`, version)
+      if (minor === 0 && patch === 0) {
```

Comment:
this will fail when `v1.5.0` (i.e the next upcoming version) gets released
```suggestion
      if (major < 1 || (major === 1 && minor === 0 && patch === 0)) {
```